### PR TITLE
add tweaks to allow buildbot to start and stop nicely on win32

### PR DESCRIPTION
**Pull request by dragon512**
This adds some small tweak to allow the same 
```python
buildbot start
buildbot stop
buildbot restart
```
commands to work on windows as it does on posix systems

http://trac.buildbot.net/wiki/Development
And especially:
http://trac.buildbot.net/wiki/SubmittingPatches

## Contributor Checklist:

* [ ] I have updated the unit tests
* [ ] I have created a file in the master/buildbot/newsfragment directory (and read the README.txt in that directory)
* [ ] I have updated the appropriate documentation

### CODE REVIEW

**Comment by dragon512** [comment]
I should note this does not support the --clean option. As a different way to communicate SIGUSR1 signal is needed as Windows does not support this.

File: .coveragerc [hunk 1]
```diff
@@ -1 +0,0 @@
-common/coveragerc
```
Comment (tardyp) [inline .coveragerc]: why did you remove that link?

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.formatting.provider": "yapf"
```
Comment (tardyp) [inline .vscode/settings.json]: I would prefer we do not commit IDE specific code here. Please add it in gitignore

File: master/docs/manual/mydashboard.html [hunk 3]
```diff
@@ -1 +0,0 @@
-../../../smokes/templates/mydashboard.html
```
Comment (tardyp) [inline master/docs/manual/mydashboard.html]: does windows not support symlinks?
Please do not remove them!
We need those symlinks so that we can test the mydashboard in the smoke tests

File: .vscode/settings.json (hunk 2 above)
Comment (dragon512) [inline .vscode/settings.json]: Ya.. that was a mistake on my end. Let me correct.

File: master/docs/manual/mydashboard.html (hunk 3 above)
Comment (dragon512) [inline master/docs/manual/mydashboard.html]: It does.. and i have them enabled for my logon. I am not sure what happened here.

**Comment by dragon512** [comment]
Fix those issue... do you have an concerns with updating the .gitignore file to ignore the .vscode directory?

**Comment by seankelly** [comment]
Editor-specific ignores should be in the person's own .gitignore file. That way it will apply to everything they work on automatically.

**Comment by dragon512** [comment]
Well I don't see a way to test it here... you need to add a windows container or something

Like I stated --clean will fail. I was thinking I could have a temp be created that the buildbot could see instead of the SIGUSER1 to have the clean shut logic work.
